lock-impl/src/main/java/com/palantir/lock/impl/LockServiceImpl.java

```diff
@@ -979,13 +957,9 @@ public LockServerOptions getLockServerOptions() {
      */
     @Override
     public void logCurrentState() {
```

Comment:
I understand if you want to maintain backcompat on this method in terms of what logging is expected.  That said, this method has way too much going on.  Any chance we could refactor it to just be delegating to other well scoped methods.  Also would be nice to remove all of it from the LockServiceImpl.  The easiest move here might be to first submit a refactor of the current logging impl and then make a smaller change to add your new logging capabilities.